.github/labeler.yml

```diff
@@ -2,4 +2,4 @@
   - .github/**/*
 
 "Issue-Docs":
-  - **/*.md
+  - ./**/*.md
```

Comment:
I'm curious about this change. The labeler appears to use:
https://github.com/isaacs/minimatch#features

What are you trying to do here and what did you see? (It's too late for me to reason through anything, but I don't mind learning)

Reply:
Initial attempt had failed.

Looking at the docs, their examples always began with "./" for root directory. I was just trying to fix the error. CI failed after this commit due to permission errors. Which was not the case before.

You might understand if you check the CI logs

Reply:
Ah. Fun https://github.com/actions/labeler/issues/12

Yeah. The spell-checker cheats. It relies on being able to run on forks and once it's merged it uses a cron to work around the fact that prs from forks are messy.

There's a very recent replacement for `pull_request`: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target -- you can try that, it might magically fix your problems.

(I haven't had the time to experiment with it.)

Reply:
In fact, while they do a *really* _poor_ job of calling it out, their recommendation includes `pull_request_target`: https://github.com/actions/labeler/blob/bab130482f3d177a5ecf31d8c2cea14fb04f8855/README.md#create-workflow

Reply:
Unfortunately I am too lazy to do these kind of thing :)

Reply:
It will be fine once we merge this in